internal/services/appservice/helpers/web_app_slot_schema.go

```diff
@@ -0,0 +1,519 @@
+}
+
+func SiteConfigSchemaLinuxWebAppSlot() *pluginsdk.Schema {
+	return &pluginsdk.Schema{
+		Type:     pluginsdk.TypeList,
+		Required: true,
+		MaxItems: 1,
+		Elem: &pluginsdk.Resource{
+			Schema: map[string]*pluginsdk.Schema{
+				"always_on": {
+					Type:     pluginsdk.TypeBool,
+					Optional: true,
+					Default:  true,
+				},
+
+				"api_management_api_id": {
+					Type:         pluginsdk.TypeString,
+					Optional:     true,
+					ValidateFunc: apimValidate.ApiID,
+				},
+
+				"api_definition_url": {
+					Type:         pluginsdk.TypeString,
+					Optional:     true,
+					ValidateFunc: validation.IsURLWithHTTPorHTTPS,
+				},
+
+				"app_command_line": {
+					Type:     pluginsdk.TypeString,
+					Optional: true,
+				},
+
+				"application_stack": linuxApplicationStackSchema(),
+
+				"auto_heal": {
+					Type:     pluginsdk.TypeBool,
+					Optional: true,
+					RequiredWith: []string{
+						"site_config.0.auto_heal_setting",
+					},
+				},
+
+				"auto_heal_setting": autoHealSettingSchemaLinux(),
+
+				"container_registry_use_managed_identity": {
+					Type:     pluginsdk.TypeBool,
+					Optional: true,
+					Default:  false,
+				},
+
+				"container_registry_managed_identity_client_id": {
+					Type:         pluginsdk.TypeString,
+					Optional:     true,
+					ValidateFunc: msiValidate.UserAssignedIdentityID,
+				},
+
+				"default_documents": {
+					Type:     pluginsdk.TypeList,
+					Optional: true,
+					Computed: true,
+					Elem: &pluginsdk.Schema{
+						Type: pluginsdk.TypeString,
+					},
+				},
+
+				"http2_enabled": {
+					Type:     pluginsdk.TypeBool,
+					Optional: true,
+					Default:  false,
+				},
+
+				"ip_restriction": IpRestrictionSchema(),
+
+				"scm_use_main_ip_restriction": {
+					Type:     pluginsdk.TypeBool,
+					Optional: true,
+					Default:  false,
+				},
+
+				"scm_ip_restriction": IpRestrictionSchema(),
+
+				"local_mysql": {
+					Type:     pluginsdk.TypeBool,
+					Optional: true,
+					Default:  false,
+				},
+
+				"load_balancing_mode": {
+					Type:     pluginsdk.TypeString,
+					Optional: true,
+					Default:  "LeastRequests",
+					ValidateFunc: validation.StringInSlice([]string{
+						"LeastRequests", // Service default
+						"WeightedRoundRobin",
+						"LeastResponseTime",
+						"WeightedTotalTraffic",
+						"RequestHash",
+						"PerSiteRoundRobin",
+					}, false),
+				},
+
+				"managed_pipeline_mode": {
+					Type:     pluginsdk.TypeString,
+					Optional: true,
+					Default:  string(web.ManagedPipelineModeIntegrated),
+					ValidateFunc: validation.StringInSlice([]string{
+						string(web.ManagedPipelineModeClassic),
+						string(web.ManagedPipelineModeIntegrated),
+					}, false),
+				},
+
+				"remote_debugging": {
+					Type:     pluginsdk.TypeBool,
+					Optional: true,
+					Default:  false,
+				},
+
+				"remote_debugging_version": {
```

Comment:
should we be clear what this refers to?
```suggestion
				"remote_debugging_visualstudio_version": {
```